gitrise.sh

```diff
@@ -86,7 +93,7 @@ process_env_vars () {
     IFS=',' read -r -a env_array <<< "$env_string"
     for i in "${env_array[@]}"
      do
-        IFS='=' read -r -a  array_from_pair <<< $i
+        IFS=':' read -r -a  array_from_pair <<< "$i"
```

Comment:
Does this mean that if I pass a URL via environment variable flag it will not work? URL contains `:`.

Reply:
if the value or key contains colon, it should be escaped using a backslash. It would then be fine.